crates/viewer/re_component_ui/src/datatype_uis/float_drag.rs

```diff
@@ -76,3 +86,17 @@ fn edit_f32_zero_to_one_raw(ui: &mut egui::Ui, value: &mut MaybeMutRef<'_, f32>)
         ui.label(re_format::format_f32(**value))
     }
 }
+
+/// Editor for a [`re_types::blueprint::components::ZoomLevel`] value from zero to 19.0.
```

Comment:
Why 0 to 19, didn't it say on zoom level it was 0 to 22?
If we decide to have this arbitrary range after all, then it should be encoded in constants

Reply:
> Why 0 to 19, didn't it say on zoom level it was 0 to 22?

See other comment. Vast topic 😅